solver/llbsolver/vertex.go

```diff
@@ -391,6 +391,8 @@ func fileOpName(actions []*pb.FileAction) string {
 			names = append(names, fmt.Sprintf("mkdir %s", a.Mkdir.Path))
 		case *pb.FileAction_Mkfile:
 			names = append(names, fmt.Sprintf("mkfile %s", a.Mkfile.Path))
+		case *pb.FileAction_Symlink:
+			names = append(names, fmt.Sprintf("symlink %s -> %s", a.Symlink.LinkPath, a.Symlink.Target))
```

Comment:
I think this arrow makes more sense with arguments in reverse order. Eg. how it is in `ls -l`.

Reply:
Clarifying, you mean you'd like to see `symlink newpath -> oldpath` (the version displayed by `ls -l`), rather than `symlink oldpath -> newpath` (the order of the arguments to `ln -s` and the `symlink` syscall)

Reply:
If the above is correct, this is done